src/EditorFeatures/CSharpTest/EditAndContinue/StatementEditingTests.cs

```diff
@@ -7060,7 +7060,8 @@ public void LocalFunction_ReadOnlyRef_ReturnType_Update()
             edits.VerifyEdits(
                 "Update [void M() { int local() { throw null; } }]@13 -> [void M() { ref readonly int local() { throw null; } }]@13");
 
-            edits.VerifyRudeDiagnostics();
+            edits.VerifyRudeDiagnostics(
+                Diagnostic(RudeEditKind.ChangingLambdaReturnType, "local", CSharpFeaturesResources.local_function));
```

Comment:
Was this a bug? or do these modifiers not have a runtime effect and we should allow them?

Reply:
This was a bug. `ref` makes the return type `int&` and `readonly` adds an attribute.